__tests__/Input.test.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Input from "@/components/common/Input";
+import { Enzyme } from "./setup.config";
+
+const { shallow } = Enzyme;
+
+describe("Input Component", () => {
+  let wrapper = shallow(
+    <Input
+      type="text"
+      placeholder="FirstName"
+      name="name"
+      value="value"
+      className="className"
+      handleChange={jest.fn()}
+    />,
+  );
+
+  it("should render 2 <div> elements", () => {
+    expect(wrapper.find("div").length).toEqual(2);
+  });
+
+  it("should render 1 <input> element", () => {
+    expect(wrapper.find("input").length).toEqual(1);
+  });
+
+  it("should allow us to set type props", () => {
+    const { type } = wrapper.props().children.props.children[0].props;
+    expect(wrapper.props());
+    expect(type).toEqual("text");
+  });
+
+  it("should allow us to set placeholder props", () => {
+    const { placeholder } = wrapper.props().children.props.children[0].props;
+    expect(placeholder).toEqual("FirstName");
+  });
+
+  it("should allow us to set name props", () => {
+    const { name } = wrapper.props().children.props.children[0].props;
+    expect(name).toEqual("name");
+  });
+
+  it("should allow us to set value props", () => {
+    const { value } = wrapper.props().children.props.children[0].props;
+    expect(value).toEqual("value");
+  });
+
+  it("should allow us to set className props", () => {
+    const { className } = wrapper.props().children.props.children[0].props;
+    expect(className).toEqual("className");
+  });
+
+  it("should allow us to set handleChange props", () => {
+    const { handleChange } = wrapper.props().children.props.children[0].props;
```

Comment:
Use jest.fn() to test that handleChange is called

Reply:
To achieve this you will need to find one of the inputs with wrapper.find() and then trigger the change event. Allowing the handleChange to be called cc @teepha